### PR TITLE
Improve the Email templates

### LINKED ISSUE #379: Improve the Email templates
**Is your feature request related to a problem? Please describe.**
We need to update the mentor acceptance/rejection emails

**Describe the solution you'd like**

This is the content for mentor acceptance email:
```
Dear [Mentor Name],
I hope this email finds you in high spirits! I am delighted to inform you that you have been selected as a mentor for ScholarX '23, and we extend our heartfelt congratulations to you!
We received a large number of qualified applicants, and after a thorough review of all candidates, we are thrilled to invite you to accept a place in our program. Your profile stood out amongst the others, and we are confident that you will contribute positively to our program.
We understand that your hard work and dedication have brought you to this moment, and we recognize your exceptional talent, experience and potential in your respective fields. We are excited to have you join our community of learners and scholars.
We look forward to seeing the unique perspective and insights you will bring to the mentees and to the program. We believe that you will flourish in this year's edition of ScholarX, and we are thrilled to be a part of your academic or professional journey.
Once again, congratulations on your selection! We cannot wait to have you on board. We will keep you informed on the next steps, and in the meantime would like to invite you to go through some of the resources that would be useful to thrive as a great mentor in ScholarX '23
```

This is the template for rejected mentors:
```
Dear [Mentor Name],
I hope this email finds you well. I wanted to take a moment to thank you for your interest in joining ScholarX '23 as a mentor and for submitting your application. We appreciate the time and effort you put into it.
After careful review of your application and considering all of the candidates, we regret to inform you that we are unable to make you part of the mentor base at this time. We received a large number of qualified applicants, and unfortunately, we could only accept a limited number of Mentors.
We understand that this news may be disappointing, and we encourage you to not be discouraged by this decision. Please know that this does not reflect on your abilities, potential or value as an individual. As you progress ahead on your academic or professional journey, we would be glad to have you as mentor for future ScholarX programs
We appreciate your interest in our program and would like to wish you all the best in your future endeavors. We are grateful for the opportunity to consider you for our program and encourage you to keep pursuing your goals and aspirations.
Thank you again for considering our program and for the time you invested in your application. We hope you find success and fulfillment in your academic and professional pursuits.
```

**Describe alternatives you've considered**
Related file: https://github.com/sef-global/scholarx/blob/scholarx-2023/src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java

**Additional context**
N/A

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #379

## Goals
update the mentor acceptance/rejection emails

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
https://github.com/sef-global/scholarx/issues/379

### CODE REVIEW

**Comment by Piumal1999** [comment]
Details: https://github.com/sef-global/scholarx/pull/380#issuecomment-1502008812

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 1]
```diff
@@ -49,15 +49,12 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
-                        "<b>Congratulations!</b><br />You have been selected by the " +
-                        "ScholarX committee to be a mentor of the " + program.get().getTitle() +
-                        " program. We will soon open up the program for students to " +
-                        "apply and keep you posted on the progress via email. Until " +
-                        "then, read more about student experience " +
-                        "<a href=\"https://medium.com/search?q=scholarx\">here</a> and reach out to us via " +
-                        "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a> " +
-                        "for any clarifications.";
+                message ="Dear" + mentor.getProfile().getName() + ",<br /><br />" +
+                        "I hope this email finds you in high spirits! I am delighted to inform you that you have been selected as a mentor for " + program.get().getTitle() + ", and we extend our heartfelt congratulations to you!" +
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "I hope this email finds you in high spirits! I am delighted to inform you that you have been selected as a mentor for " + program.get().getTitle() + ", and we extend our heartfelt congratulations to you! <br /><br />" +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 2]
```diff
@@ -49,15 +49,12 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
+                        "We received a large number of qualified applicants, and after a thorough review of all candidates, we are thrilled to invite you to accept a place in our program. Your profile stood out amongst the others, and we are confident that you will contribute positively to our program."+
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "We received a large number of qualified applicants, and after a thorough review of all candidates, we are thrilled to invite you to accept a place in our program. Your profile stood out amongst the others, and we are confident that you will contribute positively to our program.<br /><br />"+
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 3]
```diff
@@ -49,15 +49,12 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
+                        "We understand that your hard work and dedication have brought you to this moment, and we recognize your exceptional talent, experience and potential in your respective fields. We are excited to have you join our community of learners and scholars." +
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "We understand that your hard work and dedication have brought you to this moment, and we recognize your exceptional talent, experience and potential in your respective fields. We are excited to have you join our community of learners and scholars.<br /><br />" +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 4]
```diff
@@ -49,15 +49,12 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
+                        "We look forward to seeing the unique perspective and insights you will bring to the mentees and to the program. We believe that you will flourish in this year's edition of ScholarX, and we are thrilled to be a part of your academic or professional journey." +
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "We look forward to seeing the unique perspective and insights you will bring to the mentees and to the program. We believe that you will flourish in this year's edition of ScholarX, and we are thrilled to be a part of your academic or professional journey.<br /><br />" +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 5]
```diff
@@ -72,12 +69,14 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
             } else if (mentor.getState().name().equals("REJECTED")) {
 
                 message = "Dear " + mentor.getProfile().getName() + ",<br /><br />" +
-                        "Thank you very much for taking your time to apply for the " + program.get().getTitle() + " program. " +
-                        "However, due to the competitive nature of the mentor applications, your application " +
-                        "did not make it to the final list of mentors for the program. We encourage you to try " +
-                        "again next year and follow us on our social media channels for future programs. " +
-                        "If you have any clarifications, please reach out to us via " +
-                        "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a>";
+                        "I hope this email finds you well. I wanted to take a moment to thank you for your interest in joining " + program.get().getTitle() + " as a mentor and for submitting your application. We appreciate the time and effort you put into it."+
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "I hope this email finds you well. I wanted to take a moment to thank you for your interest in joining " + program.get().getTitle() + " as a mentor and for submitting your application. We appreciate the time and effort you put into it.<br /><br />"+
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 6]
```diff
@@ -72,12 +69,14 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
+                        "After careful review of your application and considering all of the candidates, we regret to inform you that we are unable to make you part of the mentor base at this time. We received a large number "+
+                        "of qualified applicants, and unfortunately, we could only accept a limited number of Mentors." +
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "of qualified applicants, and unfortunately, we could only accept a limited number of Mentors.<br /><br />" +
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 7]
```diff
@@ -72,12 +69,14 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
+                        "We understand that this news may be disappointing, and we encourage you to not be discouraged by this decision. Please know that this does not "+
+                        "reflect on your abilities, potential or value as an individual. As you progress ahead on your academic or professional journey, we would be glad to have you as mentor for future ScholarX programs"+
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "reflect on your abilities, potential or value as an individual. As you progress ahead on your academic or professional journey, we would be glad to have you as a mentor for future ScholarX programs.<br /><br />"+
```

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java [hunk 8]
```diff
@@ -72,12 +69,14 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
...
+                        "We appreciate your interest in our program and would like to wish you all the best in your future endeavors. We are grateful for the opportunity "+
+                        "to consider you for our program and encourage you to keep pursuing your goals and aspirations." +
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]:
```suggestion
                        "to consider you for our program and encourage you to keep pursuing your goals and aspirations.<br /><br />" +
```

**Review by Piumal1999 — COMMENTED** [review]
@anjula-sack shall we include these changes as well? Otherwise the email content will get cluttered.

File: src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java (hunk 8 above)
Comment (anjula-sack) [reply src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java]: yes @Piumal1999 can you commit them

**Comment by Piumal1999** [comment]
Committed the changes @anjula-sack

**Review by anjula-sack — APPROVED** [review]

**Pull request merged**